src/main/java/io/fabric8/jenkins/openshiftsync/BuildConfigToJobMapper.java

```diff
@@ -63,7 +64,9 @@ public static FlowDefinition mapBuildConfigToFlow(BuildConfig bc) throws IOExcep
 		BuildSource source = null;
 		String jenkinsfile = null;
 		String jenkinsfilePath = null;
+    GitSCM gitSCM = null;
 		if (spec != null) {
+		  LOGGER.info("Mapping BuildConfig "+bc.getMetadata().getName()+" to FlowDefinition");
```

Comment:
Requires:
- proper formatting for String concatenation (add spaces)
- null pointer protection in the log message 

if this cannot be performed, better to just display bc.getMetadata() and check bc != null